testsuites/test_lis.py

```diff
@@ -0,0 +1,23 @@
+"""Runs 'LIS-Tests.xml' using Pytest."""
+from __future__ import annotations
+
+import typing
+
+if typing.TYPE_CHECKING:
+    from target import Azure
+
+import pytest
+from lisa import LISA
+
+
+@LISA(platform="Azure", category="Functional", priority=0, area="LIS_DEPLOY")
+@pytest.mark.skip(reason="Scripts missing")
+def test_lis_driver_version(target: Azure) -> None:
```

Comment:
how about we run it on Local HyperV?

Reply:
If it's compatible with all platforms, it would just be `Target`, and if it's compatible with just a subset of platforms, its type would be `Union[Azure, HyperV]` etc.